chill-hadoop/src/main/java/com/twitter/chill/hadoop/KryoSerializer.java

```diff
@@ -47,7 +47,7 @@ public void serialize(Object o) throws IOException {
         try {
           st.writeObject(o);
           // Copy from buffer to output stream.
-          outputStream.writeInt(st.numOfWrittenBytes());
+          outputStream.writeLong(st.numOfWrittenBytes());
```

Comment:
If this is a legit use case, should probably move this to a varInt encoding, would probably be a win over our usual int in some cases.